src/crypto/olmlib.js

```diff
@@ -399,3 +400,21 @@ module.exports.pkVerify = function(obj, pubkey, userId) {
         util.free();
     }
 };
+
+/**
+ * Encode a typed array of uint8 as base64.
+ * @param {Uint8Array} uint8Array The data to encode.
+ * @return {string} The base64.
+ */
+module.exports.encodeBase64 = function(uint8Array) {
+    return Buffer.from(uint8Array).toString("base64");
```

Comment:
nb. if @uhoreg update the spec to be unpadded base64 we'll just need to trim the `=` signs off the end. Decoding could be more annoying depending on whether `Buffer` will refuse to decode strings without the correct padding.